docs/application/web/api/5.0/device_api/wearable/tizen/humanactivitymonitor.html

```diff
@@ -2750,7 +3144,7 @@ <h4>Methods</h4>
 </div>
 </div>
 <div class="interface" id="HumanActivityReadRecorderSuccessCallback">
-<a class="backward-compatibility-anchor" name="::HumanActivityMonitor::HumanActivityReadRecorderSuccessCallback"></a><h3>2.22. HumanActivityReadRecorderSuccessCallback</h3>
+<a class="backward-compatibility-anchor" name="::HumanActivityMonitor::HumanActivityReadRecorderSuccessCallback"></a><h3>2.25. HumanActivityReadRecorderSuccessCallback</h3>
 <div class="brief">
  The HumanActivityReadRecorderSuccessCallback interface is a callback interface that is invoked when recorded human activity data is successfully read.
           </div>
```

Comment:
Please change `2.23. GestureRecognitionCallback` in line 3189 into `2.26. GestureRecognitionCallback`.

Reply:
Done